gc/base/j9mm.tdf

```diff
@@ -965,3 +965,5 @@ TraceEvent=Trc_MM_SchedulingDelegate_updatePgcTimePrediction Overhead=1 Level=3
 TraceEvent=Trc_MM_IncrementalGenerationalGC_calculateConcurrentMarkWorkTime Overhead=1 Level=2 Group=resize Template="IncrementalGenerationalGC_calculateConcurrentMarkWorkTime Concurrent GMP ratio %f previous increment concurrent worktime %llu Total concurrent worktime %llu"
 
 TraceEvent=Trc_MM_SweepEndBalancedGC Overhead=1 Level=1 Group=gclogger Template="Sweep end. Duration %llu us"
+
+TraceEvent=Trc_MM_SchedulingDelegate_partialGarbageCollectCompleted_stats2 Overhead=1 Level=1 Group=kickoff Template="Evacuated %zu Eden + %zu non-Eden regions into copy-forward %zu + %zu and compact %zu regions. Eden was %zu regions."
```

Comment:
Would it better to retire existing `Trc_MM_SchedulingDelegate_partialGarbageCollectCompleted_stats` by renaming it to `Trc_MM_SchedulingDelegate_partialGarbageCollectCompleted_statsOld` and adding `Obsolete` key? In this case you can reuse `Trc_MM_SchedulingDelegate_partialGarbageCollectCompleted_stats` name

Reply:
I was gonna retire the old one in a separate change, but yes I can do this, too. I just did not know it's possible. Thanks!